### ISSUE TITLE
If you find any inaccuracies in the shapes of the letters, let me know here

### ISSUE THREAD

**Issue by jakadak** [jakadak/TF2-extended-fonts#4]

**Comment by Backslay** [comment]
Hey, I've noticed that TF2BUILD lacks some polish letters, at least in lowercase. could you fix it?

**Comment by Backslay** [comment]
Also, for some reason the time on my clock changes to letters, so there might be some numbers you replaced with letters

**Comment by Backslay** [comment]
![image](https://github.com/jakadak/TF2-extended-fonts/assets/85739476/95e34f7a-688e-4dfe-8f8a-6cebac4be93d)

Oh, and i think that those 3 letters would look better like this:

**Comment by jakadak** [comment]
Thank you @Backslay , I´ll edit them asap :))

**Comment by jakadak** [comment]
> Also, for some reason, the time on my clock changes to letters, so there might be some numbers you replaced with letters

In which font have you encountered this bug @Backslay? I checked all of my fonts and still wasn't able to find it.

> Hey, I've noticed that TF2BUILD lacks some Polish letters, at least in lowercase. Could you fix it?

The TF2 Build font never had a non-capital letter. And I'm sorry, for now, I'm not enough motivated to do that.

**Comment by Backslay** [comment]
> In which font have you encountered this bug @Backslay? I checked all of my fonts and still wasn't able to find it.

TF2BUILD, I set it as my system font, i'll attach a screenshot of the bug

> The TF2 Build font never had a non-capital letter

I mean, TF2BUILD simply uses the same letters for both upper and lowercase letters, but when installing the font it looked like the lowercase letters weren't written in TF2BUILD at all

> And I'm sorry, for now, I'm not enough motivated to do that.

No problem, you can do it whenever you want, it's your project after all

Here's the screenshot of the bug:
![image](https://github.com/jakadak/TF2-extended-fonts/assets/85739476/443339d0-19f2-4f57-985b-a5ddff7f9f29)

**Comment by Commenter25** [comment]
The lowercase t in TF2 Secondary appears to be drastically different. Is this intentional? Is there a reason for this divergence? No other letters stand out as significantly changed.

**Comment by jakadak** [comment]
Hi @Commenter25 ,

Thank you. I haven´t noticed this difference.
This spacing difference is caused by my font editor. However, I think it adds up to be more readable.
![image](https://github.com/jakadak/TF2-extended-fonts/assets/53210164/fac91ef9-5ec8-4093-bb63-0208d4dba7ef)

**Comment by jakadak** [comment]
Hello @Backslay 
This is probably due to some Windows buggy code, as the date numbers at the bottom work fine.
> Here's the screenshot of the bug: ![image](https://private-user-images.githubusercontent.com/85739476/337918906-443339d0-19f2-4f57-985b-a5ddff7f9f29.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg5Njk2MzAsIm5iZiI6MTcxODk2OTMzMCwicGF0aCI6Ii84NTczOTQ3Ni8zMzc5MTg5MDYtNDQzMzM5ZDAtMTlmMi00ZjU3LTk4NWItYTVkZGZmN2Y5ZjI5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjIxVDExMjg1MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThmNDY3YjZiNWY0MjFmYjlkMDY3MmMxNDVmYjk3ZDcyZjA3MDA1M2EyOTE5ODU1Mzc1N2QwNzMyM2FiYTkyOTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.FOJnvt7RXNmojjEOj8wYCogVOSGDbQdNYPA3shz9fgU)

**Comment by Backslay** [comment]
Dang, alright, thanks for looking into this anyway

**Comment by Commenter25** [comment]
> Thank you. I haven´t noticed this difference. This spacing difference is caused by my font editor. However, I think it adds up to be more readable.

I think there is a misunderstanding. Apologies, I should've included an image to compare.

This is the original lowercase t, straight from the game files:
![the described t](https://github.com/jakadak/TF2-extended-fonts/assets/35016761/3f199c45-c060-4098-a6d7-9a6967f32685)

This is the version in your font, which has a tail making it significantly wider: 
![the second described t](https://github.com/jakadak/TF2-extended-fonts/assets/35016761/cca04a26-ab2b-4257-97a9-b517dc80296f)

This is what I was originally concerned about. It is a very significant departure, when every other ASCII character appears to be identical.

For what it's worth, since it's been brought up up, I would disagree about the kerning in your version being more readable. The distance between f and a is slightly too close in the original, but in your version it is much too far, to the point of being distracting. It appears like there is a space between. This is the point of kerning, it's a common typography problem.

While the original font authors may have made the font very compact, it was something done intentionally. I think it is best to stay authentic to the original font where possible, and respect the authorial intent, especially if there is desire to get this merged into the game. If people using the font in other contexts are worried about the letter spacing, this is usually available to tweak manually, while still respecting kerning to appear evenly spaced.

**Comment by jakadak** [comment]
Hello @Commenter25 
I've published the TF2 secondary version with the original lowercase t letter:)
fonts/TF2Secondary_original_letter_t.ttf

I reviewed my project files and am still clueless about how the original font was created. So, I'm leaving the spacing difference for volunteers who want to contribute to this repository.
![image](https://github.com/jakadak/TF2-extended-fonts/assets/53210164/f3ac3d3c-4378-46fa-a8ce-ce7f9be660f6)

**Comment by Krasapan** [comment]
Ukrainian letters `І`, `Ї`, `Є` are invisible in-game with `TF2` font, other fonts display these letter correctly 
![20241030170302_1](https://github.com/user-attachments/assets/619223cf-09ce-4e9b-a240-2ff298e2d8df)
![20241030170438_1](https://github.com/user-attachments/assets/57598292-9430-47f3-8bcd-8edf56c6fa87)

Edited: NVM, it was the problem with the custom HUD
![20241101160015_1](https://github.com/user-attachments/assets/fdae8b43-afde-4d9d-84bd-12d114859132)

But also I should say that adding "#base (...)" to `clientscheme.res` and `fonts` folder doesn't do anything. It worked only after I replaced custom font text in `clientscheme.res` with custom font text from `cusomfontfiles.res` and replaced all font files in game's `resource` folder with yours using `custom`
